server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -113,37 +115,82 @@ private DivTag renderSummary(ActiveAndDraftQuestions activeAndDraftQuestions) {
         .withClasses(Styles.FLOAT_RIGHT, Styles.TEXT_BASE, Styles.PX_4, Styles.MY_2);
   }
 
+  private static QuestionDefinition getDisplayQuestion(QuestionCardData cardData) {
+    if (cardData.draftQuestion().isPresent()) {
```

Comment:
Elsewhere we've been using

return cardData.draftQuestion().orElseGet(cardData.activeQuestion()::get);

Reply:
Done!